x-pack/plugins/security/public/components/breadcrumb.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { coreMock } from 'src/core/public/mocks';
+
+import { Breadcrumb, BreadcrumbsProvider, createBreadcrumbsChangeHandler } from './breadcrumb';
+
+describe('security breadcrumbs', () => {
+  const setBreadcrumbs = jest.fn();
+  const { chrome } = coreMock.createStart();
+
+  beforeEach(() => {
+    setBreadcrumbs.mockReset();
+    chrome.docTitle.reset.mockReset();
+    chrome.docTitle.change.mockReset();
+  });
+
+  it('rendering one breadcrumb and it should NOT have an href attributes', async () => {
+    render(
+      <BreadcrumbsProvider onChange={createBreadcrumbsChangeHandler(chrome, setBreadcrumbs)}>
+        <Breadcrumb text={'Find'} href="/">
+          <div>{'Find'}</div>
+        </Breadcrumb>
+      </BreadcrumbsProvider>
+    );
+
+    expect(setBreadcrumbs).toHaveBeenCalledTimes(1);
+    expect(setBreadcrumbs).toHaveBeenCalledWith([{ text: 'Find' }]);
+  });
+
+  it('rendering two breadcrumb and our last breadcrumb should NOT have an href attributes', async () => {
```

Comment:
__optional nit:__ (feel free to ignore if it'd be the only reason for another CI run).
```suggestion
  it('rendering two breadcrumbs and our last breadcrumb should NOT have an href attributes', async () => {
```

Reply:
I have another PR coming up so I will fix it there to avoid another CI